lib/rubocop/runner.rb

```diff
@@ -11,9 +11,12 @@ class Runner # rubocop:disable Metrics/ClassLength
     class InfiniteCorrectionLoop < RuntimeError
       attr_reader :offenses
 
-      def initialize(path, offenses)
-        super "Infinite loop detected in #{path}."
-        @offenses = offenses
+      def initialize(path, offenses_by_iteration, loop_start = -1)
```

Comment:
These days I always use keyword params instead of optional params, but in this case it's not a big deal.

Reply:
Agreed, but I forgot to mark the constructor as `@api private`.

Reply:
Changed to `def initialize(path, offenses_by_iteration, loop_start: -1)`